### DISCUSSION TITLE
Beta 3 Scene switch camera default error

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#481] [Q&A] [answered]
Hi Y'all

Just wondering if I may have come across a bug in the latest Beta 3. 
If the WT conductor was to toggle through the various artwork for the material, it works fine for the Kingdom Hall screens but on OBS it defaults back to camera view after a media view immediately switches to another media.
Video [HERE](https://simonsheena-gmail.tinytake.com/df/13768ec/thumbnail?type=attachments&version_no=0&file_version_no=0&thumbnail_size=preview)

**Comment by e-Feitosa** [comment]
The same bug happens here too. If you click player and then stop, it works normally, if you click on an image and then click to display the next image, on the TV stage the image appears, but in the OBS Studio and in the zoom meeting the image of the camera that is filming the stage appears.

**Comment by sircharlo** [comment]
That would be a question for @mtdvlpr 😄

**Comment by mtdvlpr** [comment, marked as answer]
You are absolutely right! Oversight on my part. Thanks for finding it!

**Reply by e-Feitosa** [reply]
Just happened here in the live test at the meeting 🤣, waiting for new update. 
Thanks @mtdvlpr again for your work.
